packages/grid/_modules_/grid/hooks/features/filter/gridFilterState.ts

```diff
@@ -10,6 +10,14 @@ export const getDefaultGridFilterModel: () => GridFilterModel = () => ({
 export interface GridFilterState {
   filterModel: GridFilterModel;
 
+  /**
+   * Filtering status for each row.
+   * A row is filtered if it is passing the filters, whether its parent are expanded or not.
```

Comment:
```suggestion
   * A row is filtered if it is passing the filters, whether its parents are expanded or not.
```

Reply:
Nodes are closer to cells than mammals, they have only one parent 😄
The word you are looking for is "ancestors"